clients/src/main/java/org/apache/kafka/common/requests/FetchRequest.java

```diff
@@ -33,6 +32,7 @@
     private static final String REPLICA_ID_KEY_NAME = "replica_id";
     private static final String MAX_WAIT_KEY_NAME = "max_wait_time";
     private static final String MIN_BYTES_KEY_NAME = "min_bytes";
+    private static final String RESPONSE_MAX_BYTES_KEY_NAME = "max_bytes";
```

Comment:
Oh, I see we already had one defined below. Maybe we could just share the constant or change the other definition to `PARTITION_MAX_BYTES_KEY_NAME`?